### PR TITLE
Issue https://github.com/ATFutures/calendar/issues/55 fix :wrench:

### PR DESCRIPTION
**Pull request by serkor1**
Hi,

I took the liberty to give the code a bit of air, and more explicit usage of arguments in the `lapply()`-functions. I hope this is OK.

This PR closes https://github.com/ATFutures/calendar/issues/55 

--------

**Commit Message**

* Issue Fix: If "VALUE=DATE", "DTSTART" or "DTEND" exists in the ical-object the dates are now parsed correctly using data.frames instead of lists.

* Minor Fix: Removed redundant if-statement. `ic_dataframe()`-function already checks wether the object is a `character` or `list` and therefore the if-statement is redundant.

* Layout Fix: Increased spacing between if-statement and conditions, and added vertical space. All `lapply`-functions are now more verbose in the function calls.

### CODE REVIEW

File: R/ic_dataframe.R [hunk 1]
```diff
@@ -17,30 +17,53 @@
 #' identical(x, x_df2)
 ic_dataframe <- function(x) {
 
-  if(methods::is(object = x, class2 = "data.frame")) {
+  if (methods::is(object = x, class2 = "data.frame")) {
```
Comment (Robinlovelace) [inline R/ic_dataframe.R:20]: :+1: We could also apply an auto-formatter (maybe that's what you did).

**Review by Robinlovelace — APPROVED** [review]
Great job, many thanks @serkor1 !

File: R/ic_dataframe.R (hunk 1 above)
Comment (serkor1) [reply R/ic_dataframe.R:20]: No, I do it manually :thinking: What autoformatter are you using?

Comment (Robinlovelace) [reply R/ic_dataframe.R:20]: {styler} sometimes

Comment (Robinlovelace) [reply R/ic_dataframe.R:20]: And {styler.equals} for equals assignment.

**Pull request merged**